File: corpus/2018q3/_p2q3_ok/_check/C;0775.py
int a=27
GeraMatriz(A,a,a)
imprimeMatriz(A,a,a)
matrizTransposta(A,a,a)
maiorElemento(A,a,a)
def GeraMatriz(int Mat[][], int L, int C):
    for i=0 in range(0, L):
        for j=0 in range(0, C):
def imprimeMatriz(int M[][], int l, int c):
    for i=0 in range(0, l):
        for j=0 in range(0, c):
            print(M[i][j]+" ")
        print()
def matrizTransposta(int M[][], int l, int c):
    for i=0 in range(0, l):
        for j=0 in range(0, c):
    return Mat[l][c]
def maiorElemento(int M[][], int l, int c):
    int maior=0
    for i=0 in range(0, l):
        maior=0
        for j=0 in range(0, c):
            if M[i][j]>maior:
    return maior

File: corpus/2018q3/_p2q3_ok/_check/C;0775.java

import java.util.*;

public class Exerc3 {
    public static void main(String[] args) {
        Scanner ler = new Scanner(System.in);
        int A[][]=new int [27][27];
        int a=27;
        GeraMatriz(A,a,a);
        imprimeMatriz(A,a,a);
        matrizTransposta(A,a,a);
        maiorElemento(A,a,a);
    }
    public static void GeraMatriz(int Mat[][], int L, int C){
        for(int i=0;i<L;i++){
            for(int j=0;j<C;j++){
                Mat[i][j]=(int)(Math.random()*10);
            }
        } 
    }
    public static void imprimeMatriz(int M[][], int l, int c){
        for(int i=0;i<l;i++){
            for(int j=0;j<c;j++){
                System.out.print(M[i][j]+" ");
            }
            System.out.println();
        }
    }
    public static int matrizTransposta (int M[][], int l, int c){
        int Mat[][]=new int [l][c];
        for(int i=0;i<l;i++){
            for(int j=0;j<c;j++){
                Mat[i][j]=M[j][i];
            }
        } 
        return Mat[l][c];    
    }
    public static int maiorElemento(int M[][], int l, int c){
        int maior=0;
         for(int i=0;i<l;i++){
             maior=0;
            for(int j=0;j<c;j++){
                if(M[i][j]>maior)
                    maior=M[i][j];
            }
        } 
        return maior;
    } 

}
